apps/web/layouts/main.tsx

```diff
@@ -164,6 +165,7 @@ const Layout: NextComponentType<
             {wrapContent ? <Box width="full">{children}</Box> : children}
           </div>
         </MenuTabsContext.Provider>
+        {String(showDrawerMenu)}
```

Comment:
I'm not sure what this is doing, are we casting boolean to string and showing it on the page?

Reply:
ahhhh no good catch this was me debugging 🤦